.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release Devworkspace Operator
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'release version in format 0.y.z'
+        required: true
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Login to Quay.io
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Setup Kustomize & yq
+        run: |
+          python -m pip install --upgrade pip
+          pip install yq
+
+          cd /usr/bin
+          curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
should stick to a fixed version of the script (not master)

Reply:
ok, will lock it down to the latest 4.0.4 version